app/client/components/sectionContent.tsx

```diff
@@ -65,18 +65,28 @@ const divCss = (hasNav: boolean | undefined) => css`
   }
 `;
 
+export const SelectedTopicObjectContext = React.createContext<
+  Dispatch<SetStateAction<HelpCentreNavConfig | undefined>>
```

Comment:
The use the the name `HelpCentreNavConfig` is looking ackward to me given how we're using it. It's also not making sense to me that we pass both the id and the title of the nav in order to select (we only used the `id` in `HelpCentreNav`). Would you be able to improve these two points as part of this PR?